### DISCUSSION TITLE
[질문] 클라이언트의 PORT는 누가 어떻게 정하는가?

### DISCUSSION THREAD

**Discussion by hyeonic** [woowacourse-study/2022-http-web-basic-for-all-developer#8] [Q&A]
데이터 전송을 위해 출발지 IP, 출발지 PORT, 목적지 IP, 목적지 PORT가 필요한 것으로 알고 있어요. 목적지 IP와 PORT는 명시적으로 작성할 수 있다고 생각하지만 출발지인 클라이언트의 PORT 번호를 정하는 방법은 무엇인가요?

루나의 추천 참고 자료: [커널이 로컬포트를 선택하는 과정](https://brunch.co.kr/@alden/19)

**Comment by nbalance97** [comment]
궁금해서 찾아봤는데 아무래도 클라이언트의 포트는 무작위로 설정해주는 거 같아요😀 

https://kay0426.tistory.com/64
